README.md

```diff
@@ -1,11 +1,13 @@
-# Time Switch
+# Time Switch - A very simple timer for your needs
```

Comment:
I don't like this description. First, I wouldn't call Time Switch a *very* simple timer, given that it can execute different tasks, not just show notification like GNOME Clocks for example. Second, I don't want to pretend that I or the application knows what needs a user have: there's a list of tasks the app can execute, and only the user can decide whether it fits their needs or not. A description should be informative, not just try to "sell" the app.